### PR TITLE
dev-db/parquet_fdw: initial parquet_fdw ebuild

### PR DESCRIPTION
**Pull request by xmhd**
Initial stab at packaging parquet_fdw.

Part of WIP for [https://github.com/adjust/infrastructure/issues/10897](url)

Ebuild is currently version 9999 and thus pulls from git as parquet_fdw repo hasn't had a tag or release for quite some time. This can be set to a specific commit/date if required. Ebuild has no KEYWORDS currently until testing is complete.

Ebuild has been tested locally on a ~amd64 Gentoo Linux system. I can confirm it builds successfully.

TODO:
- Which versions of postgres is parquet_fdw compatible with?
- Non live (9999) ebuild for future release(s)
- is LICENSE 100% accurate?
- Proper description in metadata.xml
- Proper maintainer contact in metadata.xml
- Testing

### CODE REVIEW

File: dev-db/parquet_fdw/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (monsieurp) [inline dev-db/parquet_fdw/metadata.xml:1]: Copy and paste an metadata.xml file from another package in the repo.

File: dev-db/parquet_fdw/parquet_fdw-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/adjust/parquet_fdw.git"
+else
+	# temporary, no release for recent version as of 03-02-2021
+	SRC_URI="https://github.com/adjust/archive/parquet_fdw-${PV}.tar.gz"
+fi
+
+LICENSE="POSTGRESQL GPL-2"
+KEYWORDS=""
```
Comment (monsieurp) [inline dev-db/parquet_fdw/parquet_fdw-9999.ebuild:22]: amd64 at least I guess?

File: dev-db/parquet_fdw/parquet_fdw-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+# TODO: find out exactly what is compatible with this software
+POSTGRES_COMPAT=( 11 12 )
+
+inherit git-r3 postgres-multi
+
+DESCRIPTION="Parquet foreign data wrapper for PostgreSQL"
+HOMEPAGE="https://github.com/adjust/parquet_fdw"
+
+if [[ ${PV} == 9999 ]]; then
```
Comment (monsieurp) [inline dev-db/parquet_fdw/parquet_fdw-9999.ebuild:14]: Although I'm not against 9999 ebuilds, we should ask the repository owner to make a release. https://github.com/adjust/parquet_fdw is managed by @zilder I believe.

**Review by monsieurp — CHANGES_REQUESTED** [review]

**Comment by monsieurp** [comment]
Hey @zilder what's up? Could you please make a release of parquet_fdw? There's been several changes since the last one (https://github.com/adjust/parquet_fdw/tags). We need to package it for a project and having an up-to-date release would help us tremendously. Thank you!

**Comment by zilder** [comment]
Hi,
sure, here is the new release:
https://github.com/adjust/parquet_fdw/releases/tag/v0.2

Though i'm going to release a new version with major refactoring and couple of new features requested by AB team in couple of weeks.

**Comment by monsieurp** [comment]
awesome thanks @zilder :)

File: dev-db/parquet_fdw/parquet_fdw-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+	EGIT_REPO_URI="https://github.com/adjust/parquet_fdw.git"
+else
+	# temporary, no release for recent version as of 03-02-2021
+	SRC_URI="https://github.com/adjust/archive/parquet_fdw-${PV}.tar.gz"
+fi
+
+LICENSE="POSTGRESQL GPL-2"
```
Comment (zilder) [inline dev-db/parquet_fdw/parquet_fdw-9999.ebuild:21]: `parquet_fdw` is licensed under postgresql license which is bsd-like. So i think GPL isn't correct here.

File: dev-db/parquet_fdw/parquet_fdw-9999.ebuild [hunk 5]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# TODO: find out exactly what is compatible with this software
+POSTGRES_COMPAT=( 11 12 )
```
Comment (zilder) [inline dev-db/parquet_fdw/parquet_fdw-9999.ebuild:7]: I test it on 10, 11, 12 and 13

**Comment by xmhd** [comment]
@monsieurp  @zilder 

Thank you both for feedback / information. I have updated the ebuild accordingly.

**Review by monsieurp — APPROVED** [review]

**Pull request merged**